inz_irma_backend/id_card.py

```diff
@@ -0,0 +1,92 @@
+"""
+A Module to handle ID Card
+"""
+import traceback
+from datetime import datetime, timedelta
+from http import HTTPStatus
+from http.client import HTTPException
+
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+
+from inz_irma_backend import logger, InternalServerError
+
+ID_CARD_CREDENTIAL = "irma-demo.inz-id-card.idCard"
+ID_CARD_NUMBER_ATTRIBUTE = "number"
+ID_CARD_EXPIRY_DATE_ATTRIBUTE = "expiryDate"
+
+app = Flask(__name__)
+CORS(app)
+
+
+class IDCardExpiredException(Exception):
+    """
+    Expired id card exception.
+    """
+
+
+@app.errorhandler(401)
+@app.errorhandler(InternalServerError)
+@app.errorhandler(HTTPException)
+def error_handler(error):
+    """
+    Handles registered errors by returning code and description in
+    JSON format
+    """
+    response = jsonify({
+        "code": error.code,
+        "name": error.name,
+        "description": error.description,
+    })
+    return response, error.code
+
+
+@app.route('/id-card/refresh', methods=['POST'])
+def refresh_id_card_chained_session():
+    """
+    It is used a next session url in chained requests to refresh an ID Card.
```

Comment:
What do you think of adding this to the route documentation as well, so we are reminded when looking at the code?
```suggestion
    It is used a next session url in chained requests to refresh an ID Card.
    Currently a route without apiKey protection due to current irmago server limitations to use custom headers.
    We may remove or change this route if irmago accepts custom headers or we build another solution.
```